tests/unit/nupic/research/inhibition_object_test.py

```diff
@@ -39,10 +39,12 @@
 
 class InhibitionObjectTest(unittest.TestCase):
 
+  @unittest.skip("Currently fails due to switch from FDRCSpatial2 to SpatialPooler. \
```

Comment:
> Use implicit string concat rather than backslash as suggested below. Also please create an issue for adding method for getting inhibition or whatever is necessary to get this test working and put link in this comment.

Done.